language/move-prover/tests/testsuite.rs

```diff
@@ -118,108 +121,111 @@ fn test_runner(path: &Path) -> datatest_stable::Result<()> {
 }
 
 fn cvc4_deny_listed(path: &Path) -> bool {
-    let path_str = path.to_str().unwrap();
```

Comment:
I like the change here, but it's too hard to tell what content you're changing. I generally recommend splitting out this kind of change as a separate PR or at least a separate commit, so that it's easier to review. As it is, I guess I'll just trust you that the list is good

Reply:
Yeah I had to change this in order to fix a test, but as you said it would have been a lot clearer if this is a separate commit or PR. Will do next time